### PR TITLE
Feature/starboard

### LINKED ISSUE #87: Acknoledge stars on starboard tracking messages

### PR DESCRIPTION
**Pull request by DRSchlaubi**
Fixes #87

### CODE REVIEW

File: rankmigrator/build.gradle.kts [hunk 1]
```diff
@@ -58,3 +52,9 @@ tasks {
 configure<JavaPluginConvention> {
     sourceCompatibility = JavaVersion.VERSION_13
 }
+
+tasks {
+    compileKotlin {
+        kotlinOptions.jvmTarget = "12"
+    }
+}
```
Comment (Seliba) [inline rankmigrator/build.gradle.kts]: Missing newline

File: src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt [hunk 2]
```diff
@@ -106,6 +116,9 @@ class DatabaseUpdater {
             DevCordUser.findById(id)?.delete() ?: Unit
         }
     }
+
+    private fun runFaster(event: GuildMessageReceivedEvent) =
+        event.author.idLong == 238331227524956161
```
Comment (Seliba) [inline src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt]: Remove this code

File: rankmigrator/build.gradle.kts (hunk 1 above)
Comment (DRSchlaubi) [reply rankmigrator/build.gradle.kts]: lies

File: src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt (hunk 2 above)
Comment (DRSchlaubi) [reply src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt]: the code would not compile when this get's removed

**Comment by DRSchlaubi** [comment]
Why is @Seliba actually reading the code reviewers are just supposed to hit approve

File: src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt [hunk 3]
```diff
@@ -64,6 +64,7 @@ class DatabaseUpdater {
         }
         val previousLevel = user.level
         val level = transaction {
+            // If anyone asks what this does it makes your computer run faster trust me I know what I'm talking about
```
Comment (Seliba) [inline src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt]: Why is this comment here?

Comment (DRSchlaubi) [reply src/main/kotlin/com/github/seliba/devcordbot/listeners/RankSystemUpdater.kt]: Don't question it

**Comment by DRSchlaubi** [comment]
See no we have to wait for the CI just because you made me remove one comment

**Pull request merged**